server/admin/admin_server.go

```diff
@@ -39,11 +39,11 @@ func New(registry extension.Registry) *Server {
 
 // ListTrees implements trillian.TrillianAdminServer.ListTrees.
 func (s *Server) ListTrees(ctx context.Context, req *trillian.ListTreesRequest) (*trillian.ListTreesResponse, error) {
-	trees, err := s.listTreeImpls(ctx, req)
+	tl, err := s.listTreeImpls(ctx, req)
```

Comment:
Package name conflict?

TBH, I don't like this one very much, it's a strange variable name. Maybe use `resp` instead?